python/ciqueue/pytest.py

```diff
@@ -44,11 +45,6 @@ def __init__(self, index, queue):
         self.index = index
         self.queue = queue
 
-    def __len__(self):
-        # HACK: Prevent pytest from grabbing the next test
-        # TODO: Check if we could return a fake next test instead # pylint: disable=fixme
-        return 0
```

Comment:
We no longer need to hack the length to get around [this call](https://github.com/pytest-dev/pytest/blob/master/_pytest/main.py#L168) because that entire function is being overriden on line 114 below